examples/rsp-next-ts-17/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "rsp-next-ts",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": ".",
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "export": "next export",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@adobe/react-spectrum": "^3.26.0",
```

Comment:
Verified that this example app builds properly with latest RSP, but using the latest nightly and/or local verdaccio blew up in the build with errors like
```
./node_modules/@internationalized/date/dist/import.mjs
Module not found: Can't resolve '@swc/helpers/_/_class_private_field_init' in '...react-spectrum/examples/rsp-next-ts-17/node_modules/@internationalized/date/dist'

Import trace for requested module:
./node_modules/@internationalized/date/dist/import.mjs
./node_modules/@react-spectrum/calendar/dist/import.mjs
./node_modules/@adobe/react-spectrum/dist/import.mjs
```
Not exactly sure why, but confirmed that the new useId stuff works in a separate example app using react 17, next 11, and webpack 4 via https://github.com/devongovett/rsp-next